mappings/net/minecraft/client/particle/CampfireSmokeParticle.mapping

```diff
@@ -7,6 +7,7 @@ CLASS net/minecraft/class_3937 net/minecraft/client/particle/CampfireSmokePartic
 		ARG 8 velocityX
 		ARG 10 velocityY
 		ARG 12 velocityZ
+		ARG 14 lastsLonger
```

Comment:
maybe "persistent" or somethng?

Reply:
Persistent sounds like it would have something to do with IO to me
```suggestion
		ARG 14 extendedLifetime
```

Reply:
```suggestion
		ARG 14 signalFire
```
Since the block state toggle flag is called `signal_fire`? Though this name is already good enough